boxes/__init__.py

```diff
@@ -1324,7 +1324,8 @@ def mountingHole(self, x, y, d_shaft=0.0, d_head=0.0, angle=0, tabs=0):
             self.hole(x,y,d=d_shaft,tabs=tabs)
             return
 
-        self.moveTo(x + rs - self.burn, y, angle)
+        self.moveTo(x,y,angle)
```

Comment:
`rs - self.burn` moved from x to y direction. You sure about that? Also one space after commas, please!

Reply:
yesss, thats my one of my superpowers - unintentionally swap x and y coordinates ;-)

now "angle" rotates arround the center of the shaft hole...